### DISCUSSION TITLE
I don't understand...but not working for me :(

### DISCUSSION THREAD

**Discussion by ACabanero** [add2cal/add-to-calendar-button#399] [Q&A] [answered]
Hi!
Recently, I generate a new Angular project. When I add this package, I have not been able to get it to work :(.
I followed the steps in the page, but not run correctly.
![image](https://github.com/add2cal/add-to-calendar-button/assets/10090704/0b081bda-0eb6-49e8-aa2d-bca5a8ce3abd)
![image](https://github.com/add2cal/add-to-calendar-button/assets/10090704/9c39eeb0-9943-4294-9d39-89caee9eab59)

Any suggestion?

Thanks :)

**Comment by jekuer** [comment]
Indeed, strange.
What do the logs say (browser console, node)?

**Reply by ACabanero** [reply]
No logs, no errors :(
![image](https://github.com/add2cal/add-to-calendar-button/assets/10090704/9b6dcb92-7f21-49e3-be65-b4d30be1ab96)

**Comment by Zoap** [comment]
try moving the "import 'add-to-calendar-button';" into the Module instead of the Component.

**Reply by ACabanero** [reply]
Hi! It's not a solution for me :(
![image](https://github.com/add2cal/add-to-calendar-button/assets/10090704/2a16a246-3a62-405e-a78f-be843d7d9645)
![image](https://github.com/add2cal/add-to-calendar-button/assets/10090704/27d43366-aacf-4509-9297-57316cfc7667)

**Reply by jekuer** [reply]
@ACabanero , could you maybe add the "debug" option and check whether it prints something into the console?

**Reply by ACabanero** [reply, marked as answer]
Solved!!!
To solve:

1. Remove node_modules folder and package-lock.json.
2. Update package.json (dependencies) with this libraries versions in:  https://stackblitz.com/edit/add-to-calendar-button-angular?file=package.json (your example)
3. Build and launch
![image](https://github.com/add2cal/add-to-calendar-button/assets/10090704/caf65a06-c698-457c-bf0f-ea0925cdaddc)

Thanks for all :)
